test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+describe RecipesController do
+  let (:uri) {"a53ef6c8495adcb9f2859b1e5d99e9ba"}
+
+  describe "index" do
+
+    it "should get index" do
```

Comment:
There are some interesting cases you're not covering here:

- What happens if no search term is provided?
- What if the search returns no results?

It might also be worthwhile to add some tests around the paging parameters:

- What happens if they're absent?
- Do you get different results when they change?
- What if you send a bogus value, like a negative page number?